webpack.config.js

```diff
@@ -76,11 +76,32 @@ Encore
     /ca|de|es|fr|pl|pt-br/
   ))
 
+  /**
+   * added to fix:
+   * "BREAKING CHANGE: The request failed to resolve only because it was resolved as fully specified
+   * (probably because the origin is strict EcmaScript Module,
+   * e. g. a module with javascript mimetype, a '.mjs' file, or a '.js' file
+   * where the package.json contains '"type": "module"')."
+   * in @cubejs-client
+    */
+  .addRule({
+    test: /\.m?js$/,
+    resolve: {
+      fullySpecified: false,
+    },
+  })
+
   .enableStimulusBridge('./assets/controllers.json')
 
-  .enableSingleRuntimeChunk()
+  // When enabled, Webpack "splits" your files into smaller pieces for greater optimization.
   .splitEntryChunks()
```

Comment:
I am curious, does the order between `splitEntryChunks` and `enableSingleRuntimeChunk changes anything ?

Reply:
I don't think so. I just added comments and put them in the same order as in the docs: https://github.com/symfony/symfony-docs/blob/deca2b1acdb09e15432b0240731209207f8e5880/frontend/encore/installation.rst